packages/ui-dom-utils/src/getFontSize.ts

```diff
@@ -35,32 +36,34 @@ const COMPUTED_CACHE = {}
  * Gets font size in px
  *
  * @module getFontSize
- * @param {ReactComponent|DomNode} el - component or DOM node
- * @returns {Object} font size in px
+ * @param { Node | Window | React.ReactElement | function | undefined | null } el - component or DOM node
+ * @param { boolean } ignoreCache
+ * @returns { number } font size in px
  */
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'el' implicitly has an 'any' type.
-function getFontSize(el, ignoreCache) {
+function getFontSize(
+  el?: Node | Window | React.ReactElement | ((...args: any[]) => any) | null,
+  ignoreCache = false
+) {
   if (!canUseDOM) {
     return 16
   }
 
   const container = el || ownerDocument(el).documentElement
+  const containerString = container.toString()
```

Comment:
Should this be JSON.stringify() instead of toString()?

Reply:
hmm good question. As I see it was using toString(), but this is far form ideal, since it will be just some String like "Element". JSON.stringify would slow this code down. So I'd use a Map https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map , this can use objects as keys.

Reply:
Converted it to a WeakMap, thanks